fleetspeak/src/server/proto/fleetspeak_server/resource.proto

```diff
@@ -33,4 +33,9 @@ message ClientResourceUsageRecord {
 
   int32 mean_resident_memory_mib = 10;
   int32 max_resident_memory_mib = 11;
+
+  int32 mean_io_read_mib = 13;
```

Comment:
For resident memory using megabytes as a unit feels natural, since it gives us enough granularity. For mean_io_read_mib - I'm not sure that megabytes granularity is enough. I.e. if a process reads 500kb or 0 within 10 minutes - we won't be able to tell the difference.

Would it make sense to either:
- Have `kib` as the unit and use in64 as a field type?
- Keep `mib` as the unit, but use `double` as a field type?

Reply:
Following an offline discussion with @mbushkov, we'll implement it in the first way suggested - have `KiB` as the unit and use `int64` as the field type.